tap_hubspot/hubspot.py

```diff
@@ -978,6 +978,7 @@ def paginate(
         ),
         jitter=backoff.full_jitter,
         max_tries=20,
+        max_time=5 * 60,
```

Comment:
From what I understand, This will limit the time that `do` can do things. Including all the retries. Meaning that in most cases it will never retry.